config.yaml

```diff
@@ -0,0 +1,11 @@
+stats:
+  interval: 5 #second
+  api-url: ""
+jsonrpc:
+  miner:
+    url: "http://167.172.98.186:2345/rpc/v0"
+  lotus:
+    url: "http://167.172.98.186:1234/rpc/v0"
+  token-address: "/home/mak/.lotus/token"
```

Comment:
Does this need to be hardcoded? Can we use `~` so user doesn't have to take care of the config file by default?

Reply:
:+1:

Reply:
This is now resolved in code as follows, meaning that if a specific path is not defined in the config file, the default path will be `~/.lotus/token`

```go
const (
	DefaultPathTokenFile = "/.lotus/token"
)

// Read lotus token from file
// If token file address is not provided in configuration, default path will be used.
func ReadTokenFromFile() string {
	tokenFile := viper.GetString("jsonrpc.token-address")
	if tokenFile == "" {
		// set to default if not defined in config
		usr, _ := user.Current()
		dir := usr.HomeDir
		tokenFile = dir + DefaultPathTokenFile
	}
....
```